.circleci/config.yml

```diff
@@ -3,32 +3,55 @@
 # Check https://circleci.com/docs/2.0/language-ruby/ for more details
 #
 version: 2
+
+commands:
+  bundle-yarn-install:
+    steps:
+      - restore_cache:
+          key: identity-oidc-bundle-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install dependencies
+          command: bundle install --deployment --jobs=4 --retry=3 --without deploy development doc production --path vendor/bundle
+      - save_cache:
+          key: identity-oidc-bundle-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+      - restore_cache:
+          key: identity-oidc-yarn-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install Yarn
+          command: yarn install --cache-folder ~/.cache/yarn
+      - save_cache:
+          key: identity-oidc-yarn-{{ checksum "yarn.lock" }}
+          paths:
+            - ~/.cache/yarn
+  build-release:
+    steps:
+      - run:
+          name: Create deploy.json
+          command: bundle exec rake login:deploy_json
+      - run:
+          name: Copy vendor dependencies
+          command: make copy_vendor
```

Comment:
we don't deploy from CI... is this just a matter of making sure the steps work & don't blow up?

Reply:
Hm, well that's a surprise to me! 😄 There's a `deploy` task here in the CircleCI configuration, so I thought we did. I'm not very familiar with this, so I'll give a look over the handbook link you shared below and see what changes might need to be made here.

Reply:
This is addressed by #90